example/ck_tile/03_gemm/gemm_basic.hpp

```diff
@@ -54,8 +54,7 @@ using CDataType   = Types::CDataType;
 auto create_args(int argc, char* argv[])
 {
     ck_tile::ArgParser arg_parser;
-    arg_parser.insert("b", "1", "batch size")
-        .insert("m", "3840", "m dimension")
+    arg_parser.insert("m", "3840", "m dimension")
```

Comment:
do we not supporting batch (`b`) in this example?

Reply:
This is simple gemm, not batched.